### PR TITLE
i18n(zh-cn): Translated content-collections.mdx

### PR DESCRIPTION
**Pull request by allenli178**
What kind of changes does this PR include?
- New or updated content
- Translated content
Description
- What does this PR change? Give us a brief description.
Translated content-collections.mdx to Simplified Chinese.

### CODE REVIEW

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+      });
+    }
+    ```
+
+有关使用内容集合的博客教程的完整示例，请参见教程 repo 的[内容集合分支](https://github.com/withastro/blog-tutorial-demo/tree/Content-Collections)。
+
+## 用 Remark 修改 Fronmatter
+
+:::caution
+**不推荐** Remark 和 rehyp 插件访问 _raw_ Markdown 或 MDX 文档的 frontmatter。这就意味着 `remarkPluginFrontmatter` 的 frontmatter 与类型安全的 `schema` 是分开处理的，并且不会反映通过 Astro 应用的任何更改或缺省值。使用风险自负！
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/content-collections.mdx:580]:
```suggestion
**不推荐** Remark 和 rehyp 插件访问 _原始_ Markdown 或 MDX 文档的 frontmatter。这就意味着 `remarkPluginFrontmatter` 的 frontmatter 与类型安全的 `schema` 是分开处理的，并且不会反映通过 Astro 应用的任何更改或缺省值。使用风险自负！
```

File: src/content/docs/zh-cn/guides/content-collections.mdx [hunk 2]
```diff
@@ -0,0 +1,600 @@
...
+:::
+
+Astro 支持 remark 或者 rehype 插件[直接修改 frontmatter](/zh-cn/guides/markdown-content/#modifying-frontmatter-programmatically)。你可以通过使用从 `render()` 返回的 `remarkPluginFrontmatter` 属性在一个内容条目中访问这个修改过的 Frontmatter:
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/content-collections.mdx:583]: `/zh-cn/guides/markdown-content` need update

Comment (liruifengv) [inline src/content/docs/zh-cn/guides/content-collections.mdx:583]:
```suggestion
Astro 支持 remark 或者 rehype 插件[直接修改 frontmatter](/zh-cn/guides/markdown-content/#modifying-frontmatter-programmatically)。你可以通过使用从 `render()` 返回的 `remarkPluginFrontmatter` 属性在一个内容条目中访问这个修改过的 frontmatter:
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Comment by liruifengv** [comment]
This page has some broken links. Because `/zh-cn/guides/markdown-content` and `/zh-cn/reference/api-reference/` was outdated and needed update. 

I will update them later. I am working. Your PR should be merged after my update. 

Thank you for your contribution. @allenli178

**Comment by Yan-Thomas** [comment]
Thanks for this @allenli178, as @liruifengv pointed out, this page isn't available for translation yet, I will mark this as a draft and ping you when this page gets ready for translation so you can make the necessary changes to have it reviewed and merged. 🙌

**Comment by liruifengv** [comment]
@allenli178 如果你愿意帮忙翻译的话，可以在 https://i18n.docs.astro.build/ 查看翻译状态。所有 frontmatter 中标记 i18nReady 为 true 的是可以翻译的。

**Pull request closed without merging**